src/app/background/store/sagas/subscriptions.saga.ts

```diff
@@ -79,7 +79,7 @@ export function* autoSubscribeSaga() {
   if (nbSubscriptions === 0) {
     yield all(
       PRESELECTED_CONTRIBUTORS_IDS.map(contributorId =>
-        put(subscribe(contributorId))
+        put(subscribe(contributorId, { manual: false }))
```

Comment:
I think that it would be clearer to create a new action: `autoSubscribe(contributorID)`

Reply:
I think it's quite fine to use action `meta` to add precision on the origin of this subscription as first suggested in https://github.com/dis-moi/extension/issues/782#issuecomment-781576687

Reply:
> i mean you give to the action {manual: false}, but it's already by default on {manual: true}, so i ask if its still working ?

The second argument of the subscribe method is an object with a `manual` property set to `false` so it will override  `manual` default value which is `true`.